tox.ini

```diff
@@ -16,7 +16,8 @@ deps =
   -r requirements.txt
   django111: Django>=1.11.17,<2.0
   django22: Django>=2.2.6
-  backendAllauth: django-allauth
+  django3: Django>=3
+  backendAllauth: django-allauth==0.40.0
```

Comment:
Best test against the latest version of allauth, but drop support for 1.11 with allauth (as 1.11 is unsupported by allauth)

Reply:
Removed